sql-join/magic-mallrats.sql

```diff
@@ -0,0 +1,6 @@
+select "customers"."firstName", "customers"."lastName"
+from "rentals"
+join "inventory" using ("inventoryId")
+join "customers" using ("customerId")
+join "films" on "inventory"."filmId" = "films"."filmId"
```

Comment:
Did you do it this way just for practice? You could also use `using ("filmId")`, which is shorter. However, the "on" syntax is good to learn since sometimes it's the only way to do the join.

Reply:
I just thought it was necessary when not using a foreign key from your "from" table. I guess not.